lib/pbench/server/api/resources/users_api.py

```diff
@@ -296,40 +296,48 @@ class UserAPI(Resource):
     Abstracted pbench API to get user data
     """
 
+    # Helper NamedTuple containing a target user, expected http status to return with message
```

Comment:
The word "Helper" is both redundant and perhaps a bit misleading -- the named tuple doesn't "help", it's by nature a passive container. (It's really a class that instantiates objects with the ability to act as a tuple as well as an object instance with named attributes... which is a pretty cool idea, but still just a "fancy C struct".) Also not really much point in repeating the base type `NamedTuple` in the comment.

Maybe "Fancy C struct bundling target user, suggested HTTP result status and message" would be a bit blunt, though. ;-)

Reply:
I'm wondering if we can get by without a comment:  ideally, code should be self-documenting -- in this case, we're talking about a single definition (i.e., it's almost just a single line) -- and the code says, "`TargetUser` is a `NamedTuple` which contains three members, `target_user` (which is a `User`), `http_status` (which is an `HTTPStatus`), and `http_message` (which is a `str`)".

Is there really anything that we need to add to that, here?

Reply:
I agree the Named tuple is self explanatory and doesn't need a comment to re-describe it